examples/advanced/nodejs-fetch-and-websocket-options.ts

```diff
@@ -18,7 +18,7 @@ import { IncomingMessage } from "http"
 // We add the example imports here to make the example clearer, in your
 // own code, group them with yours.
 import nodeFetch from "node-fetch"
-import WebSocketClient from "ws"
+import * as WebSocketClient from "ws"
```

Comment:
```suggestion
import WebSocketClient from "ws"
```

Reply:
I went reading the 7.x changes over 6.x of `ws` library and nothing changed in regard to module nor imports/exports.

I suspect you don't have `allowSyntheticDefaultImports` enabled in your project so plain `import WebSocketClient from "ws"` doesn't work as expected.

This project defines it in top-level `tsconfig.json` (https://github.com/dfuse-io/client-js/blob/master/tsconfig.json#L16). That's why `import WebSocketClient from "ws"` works as-is.

As such, I'll not merge such change. I'll merge the typo fix however, thanks for it.